### DISCUSSION TITLE
PHP Laravel Spatie Backup Multiple Database

### DISCUSSION THREAD

**Discussion by hakkidilek** [spatie/laravel-backup#1052] [Q&A]
I'm using Laravel Spatie Backup to back up my Laravel project. In fact, the system works as follows. There is an admin panel where we activate customer accounts. While creating the customer, an automatic database is created for each customer. This created database is saved in the customer table. For examples below

Admin Database

id, user, client_db_name

1, Jhon, client_001
2, Test, client_002
3, Mike, client_003
....

Clients Database name (not table name, different database)

client_001
client_002
client_003
...

I can not get a manual backup one by one, there are 100s of client databases.

How can merge different Admin Database with All Custom Database

**Comment by drbyte** [comment]
Do any of the ideas posted in #926 or https://github.com/tenancy/multi-tenant/issues/354 help?
